integration-tests/chopsticks/src/network/polkadot.ts

```diff
@@ -0,0 +1,26 @@
+import { setupContext, SetupOption } from '@acala-network/chopsticks-testing'
+import type { Config } from './types.js'
+
+export const options: SetupOption = {
+	endpoint: ['wss://rpc.polkadot.io', 'wss://polkadot-rpc.dwellir.com', 'wss://rpc.ibp.network/polkadot'],
```

Comment:
This as well.

Reply:
[46f5119](https://github.com/KILTprotocol/kilt-node/pull/614/commits/46f511924ffda667b943584517f5d87b9a5f3bcb)